src/Http/Client/Auth/Digest.php

```diff
@@ -36,9 +36,8 @@ class Digest
      * Constructor
      *
      * @param \Cake\Http\Client $client Http client object.
-     * @param array|null $options Options list.
      */
-    public function __construct(Client $client, $options = null)
+    public function __construct(Client $client)
```

Comment:
I originally removed the Response parameter from SessionStorage but returned it later because that change could actually break something.